resources/tentacle.rb

```diff
@@ -29,6 +29,11 @@
 attribute :app_path, kind_of: String, default: 'C:\Octopus\Applications'
 attribute :trusted_cert, kind_of: String
 attribute :port, kind_of: Fixnum, default: 10_933
+attribute :polling_port, kind_of: Fixnum, default: 10_943
```

Comment:
Lazy evaluation never seems to do what I expect it to. 

This appears to work, is this what you are thinking? 

```ruby
attribute :port, kind_of: Fixnum, default: 10_933 if lazy { fancy_bool(:polling) == "False" }
attribute :port, kind_of: Fixnum, default: 10_943 if lazy { fancy_bool(:polling) == "True" }
```